plugins/keyboard-navigation/test/navigation_modify_test.mocha.js

```diff
@@ -87,6 +87,7 @@ suite('Insert/Modify', function() {
         </xml>`;
 
     defineTestBlocks();
+    window.cancelAnimationFrame = function() {};
```

Comment:
This would seem to warrant a comment.  Also: should it not be undone once the suite is over?  Failing to do so seems like it could cause problems if these were amalgamated into a larger test suite.

(Here and in `navigation_test.mocha.js`.)